release-notes/topics/aap-221.adoc

```diff
@@ -0,0 +1,46 @@
+[[aap-2.2.1-intro]]
+= Ansible Automation Platform 2.2.1
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+* Modified installer to include the *ansible-builder* image
+* Updated the `openshift-clients` to 4.10.x
+* Enabled users to use certificates located on destination nodes
+* Updated the version to `pulpcore-selinux`1.3.2
+* Updated the version to `pulp_installer` 3.18
+* Enhanced the copy process of execution environments so that they require less space in the ``/tmp` directory
+* Added an enhancement such that execution environment images can now be pulled from the automation hub only
+* Updated RSA key strength for Ansible Automation Platform certificates
+* Removed `redhat.rhv` collection from supported execution environments due to Python dependencies
+* Modified the base images of execution environments so that controller backups are executed in the container
```

Comment:
This was done in 2.1.3

Reply:
Removed enhancements 'Added an enhancement such that execution environment images can now be pulled from the automation hub only' and 'Modified the base images of execution environments so that controller backups are executed in the container' from the 2.2.1 enhancements list.